http://sourceware.org/ml/gdb-cvs/2010-06/msg00161.html

### src/gdb/ChangeLog	2010/06/24 15:17:24	1.11921
### src/gdb/ChangeLog	2010/06/24 18:24:02	1.11922
## -1,4 +1,10 @@
 2010-06-24  Jan Kratochvil  <jan.kratochvil@redhat.com>
+	    Tom Tromey  <tromey@redhat.com>
+
+	Fix GDB startup on readonly filesystem.
+	* charset.c (find_charset_names): Use PEX_USE_PIPES for pex_init.
+
+2010-06-24  Jan Kratochvil  <jan.kratochvil@redhat.com>
 	    Pedro Alves  <pedro@codesourcery.com>
 
 	Fix PR 9436.
--- src/gdb/charset.c	2010/05/13 23:53:32	1.33
+++ src/gdb/charset.c	2010/06/24 18:24:03	1.34
@@ -794,7 +794,7 @@
   set_in_environ (iconv_env, "LANGUAGE", "C");
   set_in_environ (iconv_env, "LC_ALL", "C");
 
-  child = pex_init (0, "iconv", NULL);
+  child = pex_init (PEX_USE_PIPES, "iconv", NULL);
 
   args[0] = "iconv";
   args[1] = "-l";
